### PR TITLE
Prac 05 feedback

### PR DESCRIPTION
**Pull request by Lijiayi2**
@JevanChndra
@a-rosa
@nora-zyy
@AdriantoHartono

### CODE REVIEW

**Comment by Lijiayi2** [comment]
@gzy20020626

File: prac_05/emails.py
Comment (AdriantoHartono) [inline prac_05/emails.py:1]: Good code👌

File: prac_05/hex_colours.py
Comment (AdriantoHartono) [inline prac_05/hex_colours.py:1]: Nice, the code is working well 👍

File: prac_05/state_names.py
Comment (AdriantoHartono) [inline prac_05/state_names.py:1]: Its already good, but I think you need to fix the output for CODE_TO_NAME become lined up 👌

File: prac_05/wimbledon.py
Comment (AdriantoHartono) [inline prac_05/wimbledon.py:1]: Good code, a lot of function have been used in this code 👌

File: prac_05/word_occurrences.py
Comment (AdriantoHartono) [inline prac_05/word_occurrences.py:1]: Good code, it show a output as expected 👍

Comment (nora-zyy) [reply prac_05/word_occurrences.py:1]: There are very detailed step-by-step instructions.

File: prac_05/wimbledon.py
Comment (nora-zyy) [reply prac_05/wimbledon.py:1]: The code is neat and easy to understand.

File: prac_05/emails.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Time estimate: 30 minutes
+
```
Comment (a-rosa) [inline prac_05/emails.py:2]: If you have a function, its better to also have a main function and call the main in the end of the program

File: prac_05/hex_colours.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    "BLACK": "#000000",
+    "BLANCHEDALMOND": "#FFEBCD",
+    "BLUE": "#0000FF",
+    "BLUEVIOLET": "#8A2BE2"
+}
+
+while True:
+    color_name = input("Enter a color name to look up the hexadecimal code (or press Enter to quit): ").upper()
+    if not color_name:
+        break
```
Comment (a-rosa) [inline prac_05/hex_colours.py:17]: Better to print an error message and not break

File: prac_05/state_names.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+State names in a dictionary
+File needs reformatting
+"""
+
+# TODO: Reformat this file so the dictionary code follows PEP 8 convention
+CODE_TO_NAME = {"QLD": "Queensland", "NSW": "New South Wales", "NT" : "Northern Territory", "WA" : "Western Australia",
+            "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
+print(CODE_TO_NAME)
+
+state_code = input("Enter short state: ") .upper()
```
Comment (a-rosa) [inline prac_05/state_names.py:12]: Delete the space before .upper()

File: prac_05/wimbledon.py [hunk 4]
```diff
@@ -0,0 +1,52 @@
+"""
+ estimate: 40 mins
+ actual: 60 mins
+ """
+
```
Comment (a-rosa) [inline prac_05/wimbledon.py:5]: Forgotten docstring in all functions

File: prac_05/word_occurrences.py [hunk 5]
```diff
@@ -0,0 +1,35 @@
+"""
+Time estimate: 45 minutes
+"""
+
```
Comment (a-rosa) [inline prac_05/word_occurrences.py:4]: If there is a function, better to make a main function also. However, I don't think you are asked to make a separate function here.

**Pull request merged**